lighthouse-extension/app/src/bundle-entry.js

```diff
@@ -39,30 +43,46 @@ window.runLighthouseForConnection = function(
       updateBadgeFn();
       throw err;
     });
-};
+}
 
 /**
- * @param {!RawProtocol.Port} port
+ * @param {RawProtocol.Port} port
  * @param {string} url
- * @param {!Object} options Lighthouse options.
- * @param {!Array<string>} categoryIDs Name values of categories to include.
- * @return {!Promise}
+ * @param {{flags: LH.Flags}} options Lighthouse options.
+ * @param {Array<string>} categoryIDs Name values of categories to include.
+ * @return {Promise<LH.RunnerResult|void>}
  */
-window.runLighthouseInWorker = function(port, url, options, categoryIDs) {
+function runLighthouseInWorker(port, url, options, categoryIDs) {
   // Default to 'info' logging level.
   log.setLevel('info');
   const connection = new RawProtocol(port);
-  return window.runLighthouseForConnection(connection, url, options, categoryIDs);
-};
+  return runLighthouseForConnection(connection, url, options, categoryIDs);
+}
 
 /**
  * Returns list of top-level categories from the default config.
- * @return {!Array<{title: string, id: string}>}
+ * @return {Array<{title: string, id: string}>}
  */
-window.getDefaultCategories = function() {
+function getDefaultCategories() {
   return Config.getCategories(defaultConfig);
-};
+}
 
-window.listenForStatus = function(listenCallback) {
+/** @param {(status: [string, string, string]) => void} listenCallback */
+function listenForStatus(listenCallback) {
   log.events.addListener('status', listenCallback);
-};
+}
+
+if (typeof module !== 'undefined' && module.exports) {
```

Comment:
can we just comment that this is for browserify and that it's not really executed in node? this threw me a bit :)